openrtb2/device.go

```diff
@@ -74,8 +98,9 @@ type Device struct {
 	// Type:
 	//   integer
 	// Description:
-	//   The general type of device. Refer to List 5.21.
-	DeviceType DeviceType `json:"devicetype,omitempty"`
+	//   The general type of device. Refer to List: Device Types in
+	//   AdCOM 1.0.
+	DeviceType int8 `json:"devicetype,omitempty"`
```

Comment:
Another :facepalm: 

Apparently, 2.6 desperately tries to re-use AdCom's types.

But it fails - for example here.

Because AdCom 1.0 Device has `type` field, but it's called `devicetype` here.

Reply:
And OpenRTB 2.6 Device has `os` field of type string, but adcom has the same but of type int (enum)...

Don't try to reuse objects from adcom1 in openrtb2.

The attempt is incomplete.

Reuse only enums from adcom1.